code/obj/item/storage/small_storage_parent.dm

```diff
@@ -94,7 +94,7 @@
 			SPAWN_DBG(1 DECI SECOND)
 				O.attack_hand(user)
 		else if (isitem(O) && !istype(O, /obj/item/storage) && !O.anchored)
-			src.attackby(O, user)
+			src.attackby(O, user, O.loc)
```

Comment:
I don't think `O.loc` is always gonna be a storage.

Reply:
It won't necessarily be a storage, but the code using `T` (starting line 149) does a type check so it should be fine. Kinda unfortunate that that bit didn't need changing, so it's not shown immediately on github.  

I think the code might have been written in understanding that belts and stuff need to directly transfer between each other, but not that they could have direct access to another's inventory.

Reply:
Yeah but why is this added in the first place when all the code that does storage to storage transfer is already using `S`, and when `O.loc` is pretty much guaranteed to be a mob and not a storage?

Reply:
get rekt kyle